### PR TITLE
Created package controllers with BookController, and added views

### PR DESCRIPTION
**Pull request by GabyCliff**
## Summary
- Created package controllers
- Created BookController
- Created views named greeting.html
## Trello Card
https://trello.com/c/Pa3dE5ep/17-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

File: src/main/resources/templates/greeting.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Getting Started: Serving Web Content</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+</head>
+<body>
+<p th:text="'Hello, ' + ${name} + '!'" />
+</body>
+</html>
```
Comment (Jimenarosello) [inline src/main/resources/templates/greeting.html:10]: This file seems to be a duplicate of `build/resources/main/templates/greeting.html`. Which one are you using?

**Review by Jimenarosello — CHANGES_REQUESTED** [review]
Nice job! Just one comment :) And remember to add .gradle to the gitignore file 😄

File: src/main/resources/templates/greeting.html (hunk 1 above)
Comment (Jimenarosello) [reply src/main/resources/templates/greeting.html:10]: Remember to delete the duplicated class :)

**Review by rabinovichm — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
Nice work 😊

**Pull request merged**